app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comment, class_name: 'Comment', foreign_key: 'author_id', dependent: :destroy
+  has_many :like, class_name: 'Like', foreign_key: 'author_id', dependent: :destroy
```

Comment:
- [ ] You have defined **unrequired foreign keys**. Kindly note that foreign keys associations between the _User_ and the other models (_Comment_, _Post_, and _Like_) are already defined in the _User_ model. Associations defined in the Post model should simply look like:
```ruby
# app/models/post.rb
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
  
...
```